backend/src/app.ts

```diff
@@ -168,6 +162,19 @@ app.get('/page-data/:page', async (req, res) => {
     (doc) => ({ id: doc.id, ...doc.data() } as ApartmentWithId)
   );
 
+  if (page === 'home') {
```

Comment:
Hmm so this approach works if the search bar is only on the home page. Now that we're adding it to the nav bar which appears on every page, we should try a different approach. For example, if someone directly navigates to an apartment page instead of first going to the home page, this part of the code would never be called. 

Just a thought: what if we check if the`landlords` and `apts`variables have a value and if not, we set them? Open to other ideas though!